_targets.R

```diff
@@ -11,8 +11,16 @@ pal_wetdry <- c("#002D5E", "#0C7182", "#6CB7B0", "#A7D2D8", "#E0D796", "#AF9423"
 percentile_breaks = c(0, 0.05, 0.1, 0.25, 0.75, 0.9, 0.95, 1)
 percentile_labels <- c("Driest", "Drier", "Dry", "Normal","Wet","Wetter", "Wettest")
 color_bknd <- "#F4F4F4"
+  
 text_color = "#444444"
 
+# draw label text
+flow_label <- "Flow percentile at USGS streamgages relative\nto the historic record."
+source_label <- "Data: USGS National Water Information System"
+
+# font - enabeling showtext
+showtext = TRUE
```

Comment:
While this is defined here, it's still being defined in `combine_plots`. We've also got the same sequence of showtext commands being used in multiple places - this snippet:
``` 
font_legend <- 'Noto Sans Mono'
font_add_google(font_legend)
showtext_opts(dpi = 300, regular.wt = 200, bold.wt = 700)
showtext_auto(enable = enable_showtext) 
```
Since `showtext_auto` turns the text on for all functions run after it is enabled, we really only need to run this once per session. We also still need to turn off showtext after we are done, by using `showtext_auto(enable = FALSE)` after plots are drawn. One way to address both of these needs would be to write a wrapper function around the code snippet above, so it is only written in one place. Then this could be run where you define `showtext` here to turn on the fonts, and at the end of the targets list to turn it off. By doing it this way, we don't need to worry about it being run within each function that uses it, because it will get run each session by default.

Reply:
I'm a bit confused on this. I pulled out the code snipped above to the top of my `_targets.R` script while setting `showtext_auto(enable = TRUE) `  and then at the en
<img width="343" alt="Screen Shot 2022-12-22 at 2 59 35 PM" src="https://user-images.githubusercontent.com/69014273/209239486-cda6dca1-2481-42b2-96aa-4d2798d8a845.png">
d of the pipeline, I added `showtext_auto(enable = FALSE)` but I'm getting this error

Reply:
yes, that's consistent with what I'm suggesting. I have never done this before, so I'm not totally certain what the expected behavior is, but if you didn't make edits to any of the targets then I would expect that output. May require troubleshooting to determine the best way to set this up, and will definitely be valuable to communicate to others as well